ansible/vars/icds/icds_public.yml

```diff
@@ -36,7 +36,7 @@ active_sites:
   riakcs: True
   cas_ssl: True
 
-riak_ring_size: 128
+riak_ring_size: 64
```

Comment:
it has to change now because all of the existing nodes have a ring size of 64 and new ones need to have a ring of the same size. My hunch as to why it changed is that when we were initially setting up, the riak machines were deployed with a ring size of 64, and when we went to change the config, nothing took affect because the machines had already been commissioned with the smaller ring size. even if there is no data, in order to change the ring size you have to manually delete the old ring, and nobody did that. so all of the old machines show `ring_creation_size = 128` which is the value in the config but `ring_num_partitions = 64` which is the actual value in the cluster.
from the riak docs:
```
If ring_num_partitions and ring_creation_size do not agree, that means that the ring_creation_size value was changed too late and that the proper steps were not taken to start over with a new ring.

Note: Riak will not allow two nodes with different ring sizes to be joined into a cluster.
```